integration-tests/src/test/java/gov/uk/di/ipv/cri/common/api/stepDefinitions/APISteps.java

```diff
@@ -63,4 +69,59 @@ public void remove_key(String key) throws IOException {
         map.remove(key);
         sessionRequestBody = objectMapper.writeValueAsString(map);
     }
+
+    @When("user sends a valid request to authorization end point")
+    public void user_sends_a_valid_request_to_authorization_end_point()
+            throws IOException, InterruptedException, URISyntaxException {
+        response =
+                IpvCoreStubUtil.sendAuthorizationRequest(
+                        DEV_AUTHORIZATION_URI,
+                        currentSessionId,
+                        "https://di-ipv-core-stub.london.cloudapps.digital/callback",
+                        "ipv-core-stub");
+    }
+
+    @And("a valid authorization code is returned in the response")
+    public void aValidAuthorizationCodeIsReturnedInTheResponse() throws IOException {
+        JsonNode jsonNode = objectMapper.readTree(response.body());
+        assertEquals(
+                UUID.fromString(jsonNode.get("authorizationCode").get("value").textValue())
+                        .toString(),
+                jsonNode.get("authorizationCode").get("value").textValue());
+        assertEquals(
+                "https://di-ipv-core-stub.london.cloudapps.digital/callback",
+                jsonNode.get("redirectionURI").textValue());
+        assertEquals("state-ipv", jsonNode.get("state").get("value").textValue());
+    }
+
+    @When("user sends a request to authorization end point with invalid client id")
+    public void user_sends_a_request_to_authorization_end_point_with_invalid_client_id()
+            throws URISyntaxException, IOException, InterruptedException {
+        response =
+                IpvCoreStubUtil.sendAuthorizationRequest(
+                        DEV_AUTHORIZATION_URI,
+                        currentSessionId,
+                        "https://di-ipv-core-stub.london.cloudapps.digital/callback",
+                        "INVALID-CLIENT-ID");
+    }
+
+    @When("user sends a request to authorization end point with invalid redirect uri")
+    public void userSendsARequestToAuthorizationEndPointWithInvalidRedirectUri()
+            throws URISyntaxException, IOException, InterruptedException {
+        response =
+                IpvCoreStubUtil.sendAuthorizationRequest(
+                        DEV_AUTHORIZATION_URI,
+                        currentSessionId,
+                        "https://wrong-incorrect-url/callback",
+                        "ipv-core-stub");
+    }
+
+    @And("a {string} with code {int} is sent in the response body")
```

Comment:
change this line to:
`@And("a {string} error with code {int} is sent in the response body")`